### PR TITLE
Image resizing

### PR DESCRIPTION
**Pull request by muchkairichka**
Now size of the response image is formed depending on size of response text

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "terminal.integrated.automationProfile.windows": {},
```
Comment (aloneik) [inline .vscode/settings.json]: Please add .vscode folder to .gitingore

File: bot.py [hunk 2]
```diff
@@ -45,11 +45,27 @@ def build_response_text(rate, bank):
...
 def build_response_image(text):
-    photo = Image.new("I", (600, 670), "white")
-    font = ImageFont.truetype("arial.ttf", 25)
-    drawer = ImageDraw.Draw(photo)
-    drawer.text((10, 10), text, font=font, fill='black')
-    return photo 
+    
+    width = 10000
+    height = 20000
+    starting_position = (10, 10)
```
Comment (aloneik) [inline bot.py]: Please rename this variable to something like `offset`

File: bot.py [hunk 3]
```diff
@@ -45,11 +45,27 @@ def build_response_text(rate, bank):
...
+    height = bottom - top + starting_position[1] * 2
+       
+    image = Image.new('RGB', (width, height), (255, 255, 255))
+    draw = ImageDraw.Draw(image)
+
+    draw.text(starting_position, text, font=font, fill=(0, 0, 0))
+    return image 
 
 # Start the bot
 bot.polling(none_stop=True)
```
Comment (aloneik) [inline bot.py]: Please restore empty line at the end of file

File: bot.py [hunk 4]
```diff
@@ -45,11 +45,27 @@ def build_response_text(rate, bank):
     )
         
 def build_response_image(text):
```
Comment (aloneik) [inline bot.py]: Please wrap this functionality into a class. 
It is better to have a class with ability to customize image parameters

File: bot.py [hunk 5]
```diff
@@ -45,11 +45,27 @@ def build_response_text(rate, bank):
...
+    
+    font_size = 30
+    font = ImageFont.truetype('arial.ttf', font_size)
+
+    image = Image.new('RGB', (width, height), (255, 255, 255))
+    draw = ImageDraw.Draw(image)
+
+    left, top, right, bottom = draw.multiline_textbbox(starting_position, text=text, font=font)
+
+    width = right - left + starting_position[0] * 2
```
Comment (aloneik) [inline bot.py]: Don't shadow previous values of `width`, `height`, `image` and `draw` variables

File: bot.py [hunk 6]
```diff
@@ -45,11 +45,27 @@ def build_response_text(rate, bank):
...
+    height = bottom - top + starting_position[1] * 2
+       
+    image = Image.new('RGB', (width, height), (255, 255, 255))
```
Comment (aloneik) [inline bot.py]: Use named constant or color string for defining white/black color

File: bot.py [hunk 7]
```diff
@@ -45,11 +45,27 @@ def build_response_text(rate, bank):
     )
         
 def build_response_image(text):
-    photo = Image.new("I", (600, 670), "white")
-    font = ImageFont.truetype("arial.ttf", 25)
-    drawer = ImageDraw.Draw(photo)
-    drawer.text((10, 10), text, font=font, fill='black')
-    return photo 
+    
+    width = 10000
```
Comment (aloneik) [inline bot.py]: It is not neccesary to have a separate variables for size of the temporary image

**Review by aloneik — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (muchkairichka) [reply .vscode/settings.json]: Done

File: bot.py (hunk 2 above)
Comment (muchkairichka) [reply bot.py]: Done

File: bot.py (hunk 6 above)
Comment (muchkairichka) [reply bot.py]: Done

File: bot.py (hunk 4 above)
Comment (muchkairichka) [reply bot.py]: Done

**Review by aloneik — APPROVED** [review]
ok

**Pull request merged**